lib/analytics/ga4_import/relevance_report_generator.rb

```diff
@@ -0,0 +1,18 @@
+module Analytics
+  module Ga4Import
+    class RelevanceReportGenerator
+      def self.call
+        new.call
+      end
+
+      def call
```

Comment:
This method could probably also use some tidying up/converting to a more pipeline-style approach, e.g.:

```ruby
def initialize
  @data_fetcher = DataFetcher.new
end

def call
  data_fetcher.call
    .then { |paginated_data|
      PageViewConsolidator.new(paginated_data).consolidated_page_views
    }
    .then { |consolidated_page_views|
      ElasticSearchRelevancySerialiser.new(consolidated_page_views)
    }
    .relevance
    .to_json
end

private

attr_reader :data_fetcher
```